doc/2018/2018-10-02.md

```diff
@@ -0,0 +1,171 @@
+# Hello Ubuntu Server
+
+The purpose of this communication is to provide a status update and
+highlights for any interesting subjects from the Ubuntu Server Team. If
+you would like to reach the server team, you can find us at
+the #ubuntu-server channel on Freenode. Alternatively, you can sign up
+and use the Ubuntu Server Team [mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server).
+
+## Spotlight: Cosmic Release Imminent and New Core Developer
+Cosmic is in its [final week](https://wiki.ubuntu.com/CosmicCuttlefish/ReleaseSchedule) of testing. Next week the final freeze will occur and release candidate get published. If you are interested head over to the [daily ISO page](http://cdimage.ubuntu.com/ubuntu-server/daily/current/) and give Cosmic a spin! Our developers are working to close out any sigificant bugs and documentation to add a last bit of polish to the release which is scheduled for Oct 11th.
```

Comment:
Final weeks, not week. Release is Oct 18, not the 11th.

"and documentation to add a last bit of polish to the release which is scheduled for Oct 11th."

This does not read smoothly.

Reply:
reworded.